Source/FieldSolver/SpectralSolver/SpectralHankelTransform/HankelTransform.cpp

```diff
@@ -198,44 +210,24 @@ HankelTransform::HankelForwardTransform (amrex::FArrayBox const& F, int const F_
     AMREX_ALWAYS_ASSERT(ngr >= 0);
     AMREX_ALWAYS_ASSERT(F_box.bigEnd(0)+1 >= m_nr);
 
-#ifndef AMREX_USE_GPU
-    // On CPU, the blas::gemm is significantly faster
-
     // Note that M is flagged to be transposed since it has dimensions (m_nr, m_nk)
     blas::gemm(blas::Layout::ColMajor, blas::Op::Trans, blas::Op::NoTrans,
```

Comment:
Let's add stream synchronizes before/after, since with the current `blas::Queue` constructor, this runs in its own GPU stream.